### ISSUE TITLE
Q3 2023 ethereum.org product roadmap

### ISSUE THREAD

**Issue by minimalsm** [ethereum/ethereum-org-website#10666]
*To make our work more accessible and to foster more community collaboration, our team publishes an overview of our quarterly roadmap goals. See last quarter's roadmap here: #10057.*

**Greetings fellow Ethereans!**

[Our vision with ethereum.org](https://ethereum.org/en/about/) continues to be to create the best portal for Ethereum's growing community. We’re an educational resource focused on helping onboard millions of visitors to Ethereum each month. Here's a look at our major Q3 initiatives to advance this vision.

Quick point of clarification: ***this is the roadmap for [ethereum.org](http://ethereum.org/) the website, not Ethereum the protocol***. If you’re looking for the Ethereum roadmap, check out information on [upcoming network upgrades](https://ethereum.org/en/roadmap/) and learn more about [Ethereum governance](https://ethereum.org/en/governance/).

With that out of the way, let’s dive in.

# Q3 roadmap

## 🥱 tl;dr?

Here's a high-level overview of what we're working on. Read on for more details 👀

![image](https://github.com/ethereum/ethereum-org-website/assets/62268199/be476f2a-1515-461a-9f43-4ae2094fb1da)

## 🌍 Translatathon

We're excited to launch our first-ever Translatathon in Q3—a contest aimed at translating ethereum.org's content and competing for prizes 🎁

The main goals of the Translatathon are to incentivize translations into some of the less active languages, raise awareness of the Translation program, onboard new contributors to the program, and reward some of the top contributors.

Just like a traditional hackathon, our Translatathon will have an application phase, team formation, support calls & workshops, a dedicated translation week, and a period for evaluation.

_[Join the Discord to stay up-to-date](http://ethereum.org/discord)_ 🌍

## 👋 Showing translators as contributors

Thousands of people have helped translate ethereum.org into over 50 languages. However, unlike GitHub contributors, whose contributions [show directly on the pages they contribute to](https://ethereum.org/en/developers/docs/), translator contributions are hidden away [on leaderboards](https://ethereum.org/en/contributing/translation-program/acknowledgements/) or [other translator specific pages](https://ethereum.org/en/contributing/translation-program/contributors/).

To acknowledge those contributors making ethereum.org content more accessible to non-English speakers, we will begin to highlight translators on the docs and tutorials pages they’ve helped translate.

_Want to be on one of these pages? [Join the translation program](https://ethereum.org/contributing/translation-program/)._

## 💁‍♀️ Community onboarding

Ethereum.org is made possible by thousands of enthusiastic contributors. In Q3, we'll streamline our community onboarding process, making it as effortless, engaging, and enjoyable as possible. We're currently researching potential enhancements that will make a significant impact.

Look forward to improvements across our Discord and other community platforms—it's time for a glow-up! 💅

_If you have any insights or suggestions on our onboarding materials and process, we'd love to hear them! Please share your feedback [on Discord](http://ethereum.org/discord) 🙏_

## 🤿 Design system: new hub hero, diagrams, and expandable cards

Our Design System is an ongoing initiative started in 2022. As we move forward, expect comprehensive breakdowns of our plans for each quarter.

In Q3, we'll focus on:

- Designing a new hero component for hub pages, integrating our new illustrations 👀
- Redesigning our expandable cards and FAQ components for a more user-friendly experience
- Establishing a unified design language for diagrams and infographics

_We welcome your thoughts – we'll share our progress and seek feedback in our design channel on Discord. Want to get involved? [Join us on Discord!](http://ethereum.org/discord)_

## 🖍️ Illustrations

The illustrations on ethereum.org, particularly the homepage hero image, have become a distinctive part of Ethereum's culture.

In Q3, we're working with Liam Cobb, the illustrator behind this image, to create a new collection of new illustrations for our hub pages. We're in the early stages, but Liam's initial ideas and sketches have us verrrrrrry excited 😁

_Want to see our new illustrations before everyone else? [Join our Discord](http://ethereum.org/discord)._

## 📜 Solidity lang

Solidity lang’s website(s) are getting a refresh! We begun work on this in Q2. In Q3, we’ll finalise the details and ship redesigns websites of the Solidity portal, blog, and documentation. These changes will improve the overall user experience and give the websites a modern aesthethic 💅

_Follow [Solidity Lang’s Twitter](https://twitter.com/solidity_lang) to see it drop 👀_

## 🤐 Swag boxes

This one is a secret, so can’t say much. Don’t tell @nloureiro I posted this 😆

## **🕹️ Design Interactivity**

In Q2, we evaluated how interactivity could improve our site. After identifying pages that could benefit from increased interactivity, we researched different design solutions and selected the best based on user needs and feedback.

In Q3, we're planning to design the flow and UI for a new interactive component, with the goal of adding a bit more fun and engagement to certain ethereum.org pages.

## **🕹️ Implement Interactivity**

As a stretch goal, we're aiming to also launch the new interactive component(s) in Q3. We'll likely be asking for community members to help test it as we build it out.

_If you're interested in helping us test, [join us on Discord](http://ethereum.org/discord)._

## 📊 Content auditing

In the rapidly changing Ethereum ecosystem, the relevance and accuracy of our content is crucial. Ethereum.org hosts ~300 English pages, and with the help of our community, we continuously strive to keep our content updated. As part of this 'Content Auditing' epic, we will actively review and refresh the 30 most visited pages on ethereum.org, making sure our most critical content remains comprehensive, up-to-date, and user-focused.

## 🕊️ Migrate to NextJS

In Q2, we investigated changing the web framework that ethereum.org is built on: the current setup was becoming increasingly unsuitable for both the experience of open-source developers and the number of pages ethereum.org has across all of the languages it supports.

In Q3, we'll begin to implement those changes in our codebase. Starting with an MVP and (hopefully) migrating the entire website from Gatsby to NextJS by the end of Q3 🎉

_Follow along with the progress in [the ethereum-org-next GitHub repo](https://github.com/ethereum/ethereum-org-next) 👩‍💻_

## 🔍 Glossary Tooltip

Ethereum's technical language can be a hurdle for beginners, often leading users on a multiple-page hunt to understand a single term.

This is another spillover from Q2. In Q3, we'll develop a glossary tooltip feature to allow users to access term definitions directly on the page they're reading, simplifying the learning process around complex topics.

_Interested in contributing to this feature? [Join the discussion in the #website-code channel on our Discord community.](http://ethereum.org/discord)_

## 📹 Research video dubbing

Ethereum.org links to excellent educational videos on many Web3-related topics, but until now, they've only been available in English. Technology advancements have simplified video dubbing, opening doors for us to bring these valuable resources to non-English speaking audiences.

In Q3, we'll research various tools and establish a potential process to translate video content into multiple languages.

_Have thoughts or insights on video dubbing? We'd love to hear from you. [Join the conversation in our translations channel on Discord](http://ethereum.org/discord)._

## ⛽️ Network fees page

Our developer docs' gas page gets **a lot** of traffic, but it's not beginner-friendly and doesn't answer many people's key questions: Why do I need to pay gas? How can I reduce transaction fees?

In Q3, we'll ship a network fees page tailored to beginners that gives everyone the knowledge they need to use Ethereum effectively.

_We'll share early drafts of this new page in [the #content channel on our Discord](http://ethereum.org/discord). Your feedback can help us ensure we're providing clear, useful information. [Join us there!](http://ethereum.org/discord)_

# How does that sound?

We always appreciate feedback on our roadmap - if there's something you think we should work on, please let us know! We welcome ideas and PRs from anyone in the community.

Get involved: [Learn more about contributing](https://ethereum.org/en/contributing/), [hit us up on Twitter](https://twitter.com/ethdotorg), or join the community discussions in [our Discord](https://discord.gg/CetY6Y4).

**Comment by aohycent** [comment]
Great @minimalsm

**Comment by J0rgeM** [comment]
Good roadmap

**Comment by TylerAPfledderer** [comment]
@minimalsm still interested in helping out with the mirgration to NextJS 👍

**Comment by nhsz** [comment]
> @minimalsm still interested in helping out with the migration to NextJS 👍

Hi @TylerAPfledderer, thanks for your interest! I'm currently leading the development of the migration. We started working on it a few days ago and still need some time to finish some tasks and do some planning to reach a required base ground before starting with the contributions. I'll let you know when we're ready 🙏

**Comment by TylerAPfledderer** [comment]
@nhsz great, thank you!

**Comment by corwintines** [comment]
Hell yeah @TylerAPfledderer!

**Comment by minimalsm** [comment]
Closing this out with #11386!
